tests/images/gcs-secret.txt

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
I don't see this file used anywhere in the test, do we need it?

Reply:
I've left this file as a reference for what the secret file should look like, as for GCS we need to use the json file, so this is more like an example/reference.

Reply:
We normally put our examples [here](https://github.com/kubevirt/containerized-data-importer/tree/main/manifests/example)

Reply:
Moved, forgot this in the wrong place.